tikv-control.md

```diff
@@ -300,6 +300,7 @@ DebugClient::check_region_consistency: RpcFailure(RpcStatus { status: Unknown, d
 
 > **Note:**
 >
+> - It is **NOT** recommended to use the `consistency-check` command, because it is not compatible with the TiDB Garbage Collection and may mistakenly report an error.
```

Comment:
```suggestion
> - It is **NOT** recommended to use the `consistency-check` command, because it is incompatible with the garbage collection in TiDB and might mistakenly report an error.
```